packages/server/postgres/queries/getTeamPromptResponsesByIds.ts

```diff
@@ -0,0 +1,26 @@
+import {getTeamPromptResponsesByIdsQuery, IGetTeamPromptResponsesByIdsQueryResult} from './generated/getTeamPromptResponsesByIdsQuery'
+import {JSONContent} from '@tiptap/core'
+import getPg from '../getPg'
+
+export interface TeamPromptResponse extends  Omit<IGetTeamPromptResponsesByIdsQueryResult, 'content'> {
+  content: JSONContent
+}
+
+export const mapToTeamPromptResponse = (results: IGetTeamPromptResponsesByIdsQueryResult[]): TeamPromptResponse[] => {
+  return results.map((teamPromptResponse: any) => {
+    return {
+      ...teamPromptResponse,
+      content: teamPromptResponse.content as JSONContent
+    } as TeamPromptResponse
+  })
+}
+
+export const getTeamPromptResponsesByIds = async (promptResponseIds: readonly number[]): Promise<TeamPromptResponse[]> => {
```

Comment:
+1 I'd use `teamPromptResponseIds`, I know it's a little verbose but it makes me think less